src/ObjCRuntime/Registrar.cs

```diff
@@ -186,7 +186,7 @@ public void VerifyRegisterAttribute (ref List<Exception> exceptions)
 				}
 			}
 
-			static char[] invalidSelectorCharacters = new char[] { ' ', '\t' };
+			static char[] invalidSelectorCharacters = new char[] { ' ', '\t', '?', '\\', '!', '|', '@', '"', '\'', '%', '&', '/', '(', ')', '=', '^', '[', ']', '{', '}', ',', '.', ';', '-', '\n' };
```

Comment:
Should we make this public in the 'Selector' type ? Maybe a static method (not the array itself)

That would remove the duplication (with potential drift over time) and can be useful to others, both internally (designer) and externally